src/Components/UpcomingTable/UpcomingTableFilters.tsx

```diff
@@ -145,14 +158,19 @@ export const UpcomingTableFilters: React.FunctionComponent<UpcomingTableFiltersP
     ev.stopPropagation(); // Stop handleClickOutside from handling
     setTimeout(() => {
       if (releaseMenuRef.current) {
-        const firstElement = releaseMenuRef.current.querySelector('li > button:not(:disabled)');
-        firstElement && (firstElement as HTMLElement).focus();
+        const firstElement = releaseMenuRef.current.querySelector(
+          'li > button:not(:disabled)'
+        );
+        (firstElement as HTMLElement | null)?.focus();
```

Comment:
```suggestion
        firstElement && (firstElement as HTMLElement).focus();
```

I would revert this - what we want to do is check if there is an element, then focus. What the new code does is try to focus regardless of whether it exists or not. && is like a shorthand if statement.